x/gov/client/cli/query_test.go

```diff
@@ -381,3 +383,53 @@ func (s *CLITestSuite) TestCmdQueryVote() {
 		})
 	}
 }
+
+func (s *CLITestSuite) TestCmdGeConstitution() {
```

Comment:
Fix the typo here. I think it should be put with the other test cases. Having table tests with only one test is a bit overkilled otherwise.